aws/resource_aws_imagebuilder_image_recipe.go

```diff
@@ -170,6 +170,12 @@ func resourceAwsImageBuilderImageRecipe() *schema.Resource {
 				ForceNew:     true,
 				ValidateFunc: validation.StringLenBetween(1, 128),
 			},
+			"working_directory": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				ForceNew:     true,
+				ValidateFunc: validation.StringLenBetween(1, 128),
```

Comment:
The [API Reference](https://docs.aws.amazon.com/imagebuilder/latest/APIReference/API_CreateImageRecipe.html#imagebuilder-CreateImageRecipe-request-workingDirectory) notes this parameter having a maximum length of 1024.

```suggestion
				ValidateFunc: validation.StringLenBetween(1, 1024),
```